### ISSUE TITLE
generateur en fonction d'une map

### ISSUE THREAD

**Issue by organizatsiya** [BentoBoxWorld/MagicCobblestoneGenerator#93]
On my server, I have 4 different maps:

- bskyblock_world
- bskyblock_world_nether
- bskyblock_world_the_end
- world_the_end

We can currently choose the biomes to apply the generators but we cannot define a generator for a specific map.
Would it be possible to add "world": "bskyblock_world" for example as it already exists in the addon "biomes"

thanks

**Comment by BONNe** [comment]
BentoBox MagicCobblestoneGenerator does not work in non-gamemode worlds. 
Each GameMode can have its own generators.
And normally, you cannot set nether biomes in overworlds or overworlds biomes in nether. 

So the specifying the world does nothing in the context of the addon.

**Comment by organizatsiya** [comment]
Indeed, the "biomes" parameter is not taken into account although it can be defined in the file.
For example, a generator whose biome "the_end" is defined is also activated in a "plains" biome.
Personally, I will define this as a bug.

```
  "requiredBiomes": [
    "THE_END"
  ],

```

**Comment by BONNe** [comment]
Can you export generators from the database and share it here please?

It can be done via admin GUI. There is a button: `EXPORT_DATABASE` which saves file to a `/plugins/bentobox/addons/magiccobblestonegenerator/`

**Comment by organizatsiya** [comment]
[test.txt](https://github.com/BentoBoxWorld/MagicCobblestoneGenerator/files/6173820/test.txt)

**Comment by BONNe** [comment]
https://github.com/BentoBoxWorld/MagicCobblestoneGenerator/blob/develop/src/main/resources/generatorTemplate.yml#L23-L25

**Comment by organizatsiya** [comment]
indeed, I should not have put the generators on: "defaultGenerator": true
Thanks, it works much better like that.

on the other hand, it will be interesting to add a condition for the world:
  `" world ":" bskyblock_world "`, for example
